### PR TITLE
Anya/add validations manifests controller

### PR DESCRIPTION
**Pull request by aroltsch**
connects #701

### CODE REVIEW

File: app/services/external_api/bgs_service.rb [hunk 1]
```diff
@@ -32,6 +32,12 @@ def check_sensitivity(file_number)
     end
   end
 
+  def valid_file_number?(file_number)
```
Comment (mdbenjam) [inline app/services/external_api/bgs_service.rb]: Hmm, this doesn't use any BGS stuff. Should this be a class method?

File: app/services/external_api/bgs_service.rb [hunk 2]
```diff
@@ -32,6 +32,12 @@ def check_sensitivity(file_number)
...
+    number = (file_number || "").strip
+    return true if /^\d+$/ =~ number && number.length >= 8
```
Comment (mdbenjam) [inline app/services/external_api/bgs_service.rb]: Is there any upper bound on length?

File: app/services/external_api/bgs_service.rb (hunk 1 above)
Comment (aroltsch) [reply app/services/external_api/bgs_service.rb]: a class method on the BGSService?

File: app/services/external_api/bgs_service.rb (hunk 2 above)
Comment (aroltsch) [reply app/services/external_api/bgs_service.rb]: Hmm, good question. I don't know if there is. I just followed the logic that we currently have: https://github.com/department-of-veterans-affairs/caseflow-efolder/blob/master/app/models/search.rb#L17

File: app/services/external_api/bgs_service.rb (hunk 1 above)
Comment (mdbenjam) [reply app/services/external_api/bgs_service.rb]: Yes

File: app/services/external_api/bgs_service.rb (hunk 2 above)
Comment (kierachell) [reply app/services/external_api/bgs_service.rb]: BGS tells you:
```
<message>findFlashes: FileNumber is invalid, must be 8 or 9 digits</message><
```
screen shot 2017-12-05 at 7 37 23 pm

**Pull request merged**